### PR TITLE
spaCy v1.8.2

### PR DESCRIPTION
**Pull request by honnibal**
History's a bit messy, but this should work.

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {% set name = "spacy" %}
-{% set version = "1.8.1" %}
-{% set sha256sum = "519fa96d80d2ecf1268675fe5cceb5e1ffea1fb923fbb125e13b34ae7be3808e" %}
+{% set version = "1.8.2" %}
+{% set sha256sum = "dc0af2bd53e3387d447009bf90fa278e0bcf1e32d3c09ec43e7dfc871ad40716"
```
Comment (jakirkham) [inline recipe/meta.yaml]: Missing ` %}` at EOL.

Comment (rolando) [inline recipe/meta.yaml]: Closing tag is missing `%}`.

Comment (jakirkham) [inline recipe/meta.yaml]: Sorry to steal your thunder so to speak.

Comment (rolando) [inline recipe/meta.yaml]: Hey @jakirkham, no problem. The more eyes the better.

**Comment by rolando** [comment]
Rebase is required. However, there is another PR https://github.com/conda-forge/spacy-feedstock/pull/12

**Comment by rolando** [comment]
Fixed by #12.

**Pull request closed without merging**